app/target/target_test_suite.tsx

```diff
@@ -29,7 +29,8 @@ export default class TargetTestSuiteComponent extends React.Component<Props> {
                     <div className="test-case-message">
                       {child.getAttribute("message")} {child.getAttribute("type")}
                     </div>
-                    {!!child.textContent?.trim() && (
+                    // only render text content if it exists and it's not a Playwright attachment
```

Comment:
this is a little quirky--the actual node here is a `system-out` node, right?  i think the assumption we were making is that nobody actually uses `system-out` at the `testcase` level, which is funny.  it might be better to actually detect that and treat it in a special way, since i don't think it will have `message` or `type` defined.

Reply:
i guess really what you would want to be trimming is CDATA? though i'm not even sure the junit schema requires these weird variable definitions like ATTACHMENT to even be in CDATA